content/_changelogs/9.4.0.md

```diff
@@ -0,0 +1,28 @@
+## 9.4.0
+
+_Released 1/31/2022_
+
+**Features**
+
+- Enhancements were made to `.selectFile()` feedback receiving after it's
+  initial release in [9.3.0](/guides/references/changelog#9-3-0).
```

Comment:
You make the same mistake with "its" and "it's" that I do. We're doomed between the two of us now. 😅 

```suggestion
- Enhancements were made to `.selectFile()` after receiving feedback after its
  initial release in [9.3.0](/guides/references/changelog#9-3-0).
```

Reply:
Oh no! 😢